app/javascript/packs/sdk.js

```diff
@@ -18,14 +18,23 @@ const runSDK = ({ baseUrl, websiteToken }) => {
   }
 
   const chatwootSettings = window.chatwootSettings || {};
+  const localeWithVariation = window.navigator.language.replace('-', '_');
+  const defaultLocale = chatwootSettings.locale || 'en';
+
+  const locale =
+    localeWithVariation && chatwootSettings.useBrowserLanguage
+      ? localeWithVariation
+      : defaultLocale;
```

Comment:
```suggestion
  let locale = chatwootSettings.locale || 'en';
  if (chatwootSettings.useBrowserLanguage) {
    locale = window.navigator.language.replace('-', '_');
  }
```

Reply:
@nithindavid Do we need to check whether `window.navigator.language.replace('-', '_')` exist or not? It is available in all browsers, right?

Reply:
As seen here https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language

> Examples of valid language codes include "en", "en-US", "fr", "fr-FR", "es-ES", etc.

Reply:
`localeWithVariation &&` - This check is not required, right?